src/Apps/Artwork/Components/__tests__/PricingContext.test.tsx

```diff
@@ -13,6 +13,7 @@ import {
   PricingContext,
   PricingContextFragmentContainer,
 } from "../PricingContext"
+import { flushPromiseQueue } from "Utils/flushPromiseQueue"
```

Comment:
I'm pretty sure this is only used in our tests. Mind moving to our [`DevTools`](https://github.com/artsy/reaction/tree/master/src/DevTools) folder?

Reply:
Sure thing.

Reply:
Moved